node/src/identity/hsmsigners.ts

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2021 IBM All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as pkcs11js from 'pkcs11js';
+import * as elliptic from 'elliptic';
+import BN = require("bn.js");
+const ecSignature = require('elliptic/lib/elliptic/ec/signature.js');  // eslint-disable-line
```

Comment:
Is there a public API that can be used rather than poking at internals? The SDKs have been broken before by using internals of dependencies that change when we pull in newer versions.

You might be able to lose the `.js` on the end of the required module, and generally prefer `import ... from ...` instead of `require`.